### PR TITLE
Add private websocket for...

### PR DESCRIPTION
**Pull request by go-numb**
I forked "gogmocoin" and added "private websocket" functionality and wrote test code.Thank you for something very informative. I have added the code in a very small.
As for functionality, copy public websocket and add Get and Delete WebsocketAccessToken for Subscribe/Unsubscribe.

### CODE REVIEW

**Comment by ijufumi** [comment]
Thanks for your contribution!
I'll check your code later.

File: README.md [hunk 1]
```diff
@@ -28,7 +28,13 @@ https://api.coin.z.com/docs
 * `/private/v1/cancelOrder`
 * `/private/v1/closeOrder`
 * `/private/v1/closeBulkOrder`
-* `/private/v1/changeLosscutPrice`
```
Comment (ijufumi) [inline README.md:31]: You deleted this line.
But I think it is accident, don't you?

File: README.md [hunk 2]
```diff
@@ -95,5 +101,37 @@ log.Printf("ordersRes:%+v", ordersRes)
 
 [Examples](https://github.com/ijufumi/gogmocoin-examples/tree/master/app/private)
 
+
+### Private Websocket API
+```golang
+    // periodic execution
+    isPeriodic := false
+	client := summary.New(isPeriodic)
```
Comment (ijufumi) [inline README.md]: Could you please align the indentation at the beginning of the code?

File: api/private/access_token.go [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+		res, err = c.con.Delete(req, "/v1/ws-auth")
+
+	default:
+		return "", fmt.Errorf("method:%s is not supported", method)
+	}
+	if err != nil {
+		return "", err
+	}
+
+	result := new(map[string]interface{})
```
Comment (ijufumi) [inline api/private/access_token.go]: I think it is better to prepare the specific model for this response.

**Review by ijufumi — COMMENTED** [review]
Thanks for your contribution!
I think almost is OK.
But I put some comments.
Could you please confirm these comments when your freetime?

Regards.

**Comment by go-numb** [comment]
at your commands

**Review by ijufumi — APPROVED** [review]
Thank you for your amendment.
It's LGTM!

**Pull request closed without merging**